pkgs/development/compilers/gnu-cim/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchurl
+, fetchzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnu-cim";
+  version = "5.1";
+
+  outputs = ["out" "lib" "man" "info"];
+
+  src = fetchzip {
+    url = "mirror://gnu/cim/cim-${version}.tar.gz";
+    hash = "sha256-2XNYEE1Osh/DwGlKFyMyTtnXKacd7WCy27xUbIOd910=";
+  };
+
+  postPatch = ''
+    for fname in lib/{simulation,simset}.c; do
+      substituteInPlace "$fname" \
+        --replace \
+          '#include "../../lib/cim.h"' \
+          '#include "../lib/cim.h"'
+    done
```

Comment:
Is this an upstream issue?

Reply:
I believe this is simply because they build in `$(distdir)/_build` while `stdenv` does not.